ED/protocol.html

```diff
@@ -923,7 +923,7 @@ <h3 property="schema:name">Resource Representations</h3>
           </section>
 
           <section id="notifications" inlist="" rel="schema:hasPart" resource="#notifications">
```

Comment:
```suggestion
          <section id="linked-data-notifications" inlist="" rel="schema:hasPart" resource="#linked-data-notifications">
```

Alternatively, we introduce section on "notifications" and then have linked-data-notifications and live-updates as subsections. This can be decided later.